docs/csharp/programming-guide/concepts/expression-trees/index.md

```diff
@@ -104,7 +104,7 @@ Console.WriteLine(factorial);
 // Prints 120.  
 ```  
   
- For more information, see [Generating Dynamic Methods with Expression Trees in Visual Studio 2010](http://go.microsoft.com/fwlink/?LinkId=169513), which also applies to later versions of Visual Studio.
+ For more information, see [Expression Trees (C#)](../../../../csharp/programming-guide/concepts/expression-trees/index.md), which also applies to later versions of Visual Studio.
```

Comment:
:smile: ... For more information, *re-read this topic!*

![draith2](https://cloud.githubusercontent.com/assets/1622880/24434964/2ca661c2-13f8-11e7-9d3e-9763b57c147f.jpg)


"Tell Dexeter we've come full circle!" - First Decider Draith (Leonard Maguire) Dr. Who: Full Circle, 1980

How about this one? https://msdn.microsoft.com/en-us/library/mt654263.aspx